backend/models.py

```
@@ -0,0 +1,60 @@
+"""
+Pydantic models for API request/response validation
+"""
+
+from pydantic import BaseModel
+from typing import List, Optional
```

Ruff (0.16.0)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)